src/test/java/com/wikia/webdriver/common/core/drivers/browsers/HtmlUnitBrowser.java

```diff
@@ -0,0 +1,24 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+
+public class HtmlUnitBrowser extends BrowserAbstract {
+
+  @Override
+  public void setOptions() {
+
+  }
+
+  @Override
+  public WikiaWebDriver create() {
+    return new WikiaWebDriver(new HtmlUnitDriver(), false);
+  }
+
+  @Override
+  public void addExtension(String extensionName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1186)